docs/QUICKSTART.md

```diff
@@ -1,76 +1,195 @@
-
-```bash
-$ kubectl describe ImageList imagelist
-...
-Status:
-  Failed:     0
-  Success:    3
-  Timestamp:  2022-02-25T23:41:55Z
-...
-```
-
-By default, successful jobs will be deleted after a period of time. You can change this behavior by setting the following flags in the eraser-controller-manager:
-
-- `--job-cleanup-on-success-delay`: Seconds to delay job deletion after successful runs. 0 means no delay. Defaults to `0`.
-- `--job-cleanup-on-error-delay`: Seconds to delay job deletion after errored runs. 0 means no delay. Defaults to `86400` (24 hours).
-- `--job-success-ratio`: Ratio of successful/total runs to consider a job successful. 1.0 means all runs must succeed. Defaults to `1.0`.
\ No newline at end of file
+  Creating an imagelist should trigger an [ImageJob](api/v1alpha1/imagejob_types.go) that will deploy [eraser](../pkg/eraser/eraser.go) pods on every node to perform the removal given the list of images.
+
+  ```bash
+  $ kubectl get pods -n eraser-system
+  eraser-system        eraser-controller-manager-55d54c4fb6-dcglq   1/1     Running   0          9m8s
+  eraser-system        eraser-kind-control-plane                    1/1     Running   0          11s
+  eraser-system        eraser-kind-worker                           1/1     Running   0          11s
+  eraser-system        eraser-kind-worker2                          1/1     Running   0          11s
+  ```
+
+  - Pods will run to completion and the images will be removed.
+
+  ```bash
+  $ kubectl get pods -n eraser-system
+  eraser-system        eraser-controller-manager-6d6d5594d4-phl2q   1/1     Running     0          4m16s
+  eraser-system        eraser-kind-control-plane                    0/1     Completed   0          22s
+  eraser-system        eraser-kind-worker                           0/1     Completed   0          22s
+  eraser-system        eraser-kind-worker2                          0/1     Completed   0          22s
+  ```
+
+  The `imagelist` custom resource status field will contain the status of the last job.
+
+  ```bash
+  $ kubectl describe ImageList imagelist
+  ...
+  Status:
+    Failed:     0
+    Success:    3
+    Timestamp:  2022-02-25T23:41:55Z
+  ...
+  ```
+
+  By default, successful jobs will be deleted after a period of time. You can change this behavior by setting the following flags in the eraser-controller-manager:
+
+  - `--job-cleanup-on-success-delay`: Seconds to delay job deletion after successful runs. 0 means no delay. Defaults to `0`.
+  - `--job-cleanup-on-error-delay`: Seconds to delay job deletion after errored runs. 0 means no delay. Defaults to `86400` (24 hours).
+  - `--job-success-ratio`: Ratio of successful/total runs to consider a job successful. 1.0 means all runs must succeed. Defaults to `1.0`.
+
+
+6. Verify the unused images are removed
+
+    ``` shell
+    docker exec <nodeName> ctr -n k8s.io images list | grep nginx
```

Comment:
Suggest using explicit node name "kind-worker" as you did above to be consistent.  (Don't assume user will automatically know what to substitute for <nodename>)
docker exec kind-worker ctr -n k8s.io images list | grep nginx

Reply:
updated in [83eaf48](https://github.com/Azure/eraser/pull/186/commits/83eaf483c7b3f4e7463e0f3a438e51c3ddaed5f9)